src/splitInteger.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable strict */
```

Comment:
The provided code is a comment block and doesn't contain the actual `splitInteger` function implementation. Please check and add the implementation of the function to complete the task. 😊